requirements-dev.txt

```diff
@@ -8,3 +8,4 @@ sphinx==4.0.*
 sphinx-autobuild==2021.3.*
 ptvsd==4.3.*
 jupyter==1.0.*
+scipy
```

Comment:
Shouldn't the version be pinned here? Also, adding this dependency here seems unnecessary since it's being added in the next commit to the core `requirements.txt`

Reply:
I've pinned the version. I think it is better to be explicit about the requirements. Especially because `scipy` is being used for different things in the unit tests than what it is being used for in `core`. This way the unit tests won't break if `scipy` ever gets removed from `core`'s dependencies.